scorekeeper/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './App.css';
+import Dashboard from './dashboard/Dashboard'
+import Display from './display/Display'
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      strikes: 0,
+      balls: 0,
+      fouls: 0,
+      hits: 0
+    }
+  }
+
+  addStrike = () => {
+    if (this.state.strikes < 3) {
+      this.setState(prevState => {
+        return {strikes: prevState.strikes + 1}
+      })
+    } else {
+      this.setState({
+        strikes: 0,
+        balls: 0,
+        fouls: 0,
+        hits: 0
+      })
+    }
+  }
+
+  addFoul = () => {
+    if (this.state.strikes < 2){
+      this.setState(prevState => {
+        return {
+          fouls: prevState.fouls + 1,
+          strikes: prevState.strikes + 1,
+        }
+      })
+    }
+  }
+
+  addBall = () => {
+    if (this.state.balls < 4) {
+      this.setState(prevState => {
+        return {balls: prevState.balls + 1}
+      })
+    } else {
+      this.setState({
+        strikes: 0,
+        balls: 0,
+        fouls: 0,
+        hits: 0
+      })
```

Comment:
this state object seems to be repeated very much; I'd suggest either using a method for this (i.e. `resetState`), or to use an object for this; i.e.:
```javascript
const defaultState = {
  strikes: 0,
  balls: 0,
  fouls: 0,
  hits: 0
};

class App // ...
  constructor(props){
    super(props);
    this.state = { ...defaultState };
  }

  addStrike = () => {
    // ...
    else {
      this.setState({ ...defaultState });
    }
  }

  // ...

```